CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ if (HALST_STANDALONE)
     FetchContent_Declare(
         emil
         GIT_REPOSITORY https://github.com/philips-software/amp-embedded-infra-lib.git
-        GIT_TAG        4dc5736b2814734a66a1de75d2eabb356d1ed6ff # unreleased
+        GIT_TAG        feature/extend-gpio-interrupt-interface-to-enable-immediate-interrupt-handlers # unreleased
```

Comment:
Most probably superfluous, but as a reminder: update after merging the pull request in amp-embedded-infra-lib